app/Models/EntryDAO.php

```diff
@@ -726,23 +726,24 @@ private function sqlListWhere($type = 'a', $id = '', $state = FreshRSS_Entry::ST
 		$values = array();
 		switch ($type) {
 		case 'a':
-			$where .= 'f.priority > 0 ';
-			$joinFeed = true;
+			$where .= 'f.priority > ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
 			break;
 		case 's':	//Deprecated: use $state instead
-			$where .= 'e.is_favorite=1 ';
+			$where .= 'f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
+			$where .= 'AND e.is_favorite=1 ';
 			break;
 		case 'c':
-			$where .= 'f.category=? ';
+			$where .= 'f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
+			$where .= 'AND f.category=? ';
 			$values[] = intval($id);
-			$joinFeed = true;
 			break;
 		case 'f':
-			$where .= 'e.id_feed=? ';
+			$where .= 'f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' ';
```

Comment:
When we query a feed by ID, we should not filter by feed priority, don't you think?

Reply:
That's true. I removed it.